docs/how-to-guides/streaming-source-ingestion.md

```diff
@@ -35,6 +35,8 @@ stream_source = KafKaSource(name="kafkaStreamingSource",
                             )
 ```
 
+You may need to produce data and send them into Kafka as this data source in advance. Please check [Kafka data source producer](../samples/kafka_producer.py) as a reference. Also you should keep this producer running which means there are data stream keep coming into Kafka while calling the 'materialize_features' below.
```

Comment:
Can we direct them to this file? test\prep_azure_kafka_test_data.py

Reply:
Modified the link to '' test\prep_azure_kafka_test_data.py" and also added a loop in the test script to keep it producing data stream.